cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/OpenAI.scala

```diff
@@ -45,6 +45,45 @@ trait HasPrompt extends HasServiceParams {
   def setPromptCol(v: String): this.type = setVectorParam(prompt, v)
 }
 
+trait HasBulkPrompt extends HasServiceParams {
+  val bulkPrompt: ServiceParam[Seq[String]] = new ServiceParam[Seq[String]](
+    this, "bulkPrompt", "Sequence of prompts to complete", isRequired = false)
+
+  def getBulkPrompt: Seq[String] = getScalarParam(bulkPrompt)
+
+  def setBulkPrompt(v: Seq[String]): this.type = setScalarParam(bulkPrompt, v)
+
+  def getBulkPromptCol: String = getVectorParam(bulkPrompt)
+
+  def setBulkPromptCol(v: String): this.type = setVectorParam(bulkPrompt, v)
+}
+
+trait HasIndexPrompt extends HasServiceParams {
+  val indexPrompt: ServiceParam[Seq[Int]] = new ServiceParam[Seq[Int]](
+    this, "index", "Sequence of indexes to complete", isRequired = false)
+
+  def getIndexPrompt: Seq[Int] = getScalarParam(indexPrompt)
+
+  def setIndexPrompt(v: Seq[Int]): this.type = setScalarParam(indexPrompt, v)
+
+  def getIndexPromptCol: String = getVectorParam(indexPrompt)
+
+  def setIndexPromptCol(v: String): this.type = setVectorParam(indexPrompt, v)
+}
+
+trait HasBulkIndexPrompt extends HasServiceParams {
+  val bulkIndexPrompt: ServiceParam[Seq[Seq[Int]]] = new ServiceParam[Seq[Seq[Int]]](
+    this, "bulkIndex", "Sequence of index sequences to complete", isRequired = false)
```

Comment:
The second arg of the params should match the name of the param so this would be 
```suggestion
    this, "bulkIndexPrompt", "Sequence of index sequences to complete", isRequired = false)
```